sklearn/covariance/_graph_lasso.py

```diff
@@ -78,29 +78,6 @@ def alpha_max(emp_cov):
     return np.max(np.abs(A))
 
 
-class _DictWithDeprecatedKeys(dict):
-    """Dictionary with deprecated keys.
-
-    Currently only be used in GraphicalLassoCV to deprecate keys"""
```

Comment:
This was only used in GraphicalLassoCV to be able to have deprecated keys. Since it's not used anymore I removed it, but we could also move it in utils.deprecation if we think that it's good to keep it around. What do you think ?

Reply:
I like `_DictWithDeprecatedKeys` but I think we should not maintain dead code but probably WANGI.

Are the odds of reusing it high?

Reply:
I'm okay with removing this code.